models/safe/arbitrum/safe_arbitrum_transactions.sql

```diff
@@ -0,0 +1,57 @@
+{{ 
+    config(
+        materialized='incremental',
+        alias='safe_arbitrum_transactions',
```

Comment:
i'd recommend making alias just `transactions`, otherwise your output will be:
`safe_arbitrum.safe_arbitrum_transactions`

this will apply across all your open PRs for transactions, as i'm looking all at once :)

Reply:
i'll knock these out real quick, i'm in process of cleaning out spellbook queue, no problem

Reply:
That shorter alias makes su much more sense, I wasn't thinking about hte implications. Thanks a lot!